docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:20
+
+ARG PORT=3000
+
+ENV NODE_ENV=production
+
+WORKDIR /src
+
+COPY . .
+RUN npm install --production=false
```

Comment:
I'll try and tidy this up a bit - basically prisma is a dev dependency which you don't get with a production install, but actually it would probably be better to install that separately and not get all non devDeps that we don't need.

Reply:
OK, I've turned production back on and moved prisma install onto a separate line, but i did find I needed to make nuxt test-utils and eslint into production dependencies because they're references as modules in nuxt.config .ts. Which seems a bit weird, but not sure there's any way round it.